core/tailwind.config.js

```diff
@@ -1,6 +1,10 @@
 /** @type {import('tailwindcss').Config} */
 const config = {
-  content: ['./app/**/*.{ts,tsx}', './components/**/*.{ts,tsx}'],
+  content: [
+    './app/**/*.{ts,tsx}',
+    './components/**/*.{ts,tsx}',
+    '!./node_modules/**', // Exclude everything in node_modules to speed up builds
```

Comment:
Interesting, do you know why this happens? I don't understand why we have to exclude a path we aren't including. 🤔

Reply:
I wish I knew!

Reply:
Seems like it's expected: https://github.com/tailwindlabs/tailwindcss/issues/5014#issuecomment-1047038219